### DISCUSSION TITLE
pannello controllo classi scrutinate

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#527] [Idee e proposte]
Buonasera ,per una comoda e più sbrigativa procedura scrutinale ,sarebbe possibile integrare un panello relativo agli scrutini dove si vede nell'immediato le classi scrutinate (segnate in verde con una spunta) e le classi da scrutinare (segnate in rosso con una x)?

**Comment by trinko** [comment]
La segno come richiesta.
